docs/Model_Assumptions.rst

```diff
@@ -21,23 +21,41 @@ Non-dispatchable sources of generation
 
 `Examples`:
 
-    - PV plant
-    - Wind plant
+    - PV plants
```

Comment:
Note that only `Energy Production` is fully capitalized: 
![grafik](https://user-images.githubusercontent.com/44204527/109810670-1f44b800-7c2a-11eb-9864-c42dd2d8a5d3.png)

Reply:
thanks for noticing!